src/routes/+page.svelte

```diff
@@ -203,6 +270,104 @@
   {$_("contact_us")}
 </h1>
 
+<div class="mb-20 flex flex-col items-center gap-2">
+  <form
+    class="w-full max-w-xl"
+    method="POST"
+    on:submit|preventDefault={handleSubmit}
+  >
+    <div class="mb-6 md:flex md:items-center">
+      <div class="md:w-1/3">
+        <label
+          class="mb-1 block pr-4 font-bold text-gray-500 md:mb-0 md:text-right"
+          for="name"
+        >
+          Your name
```

Comment:
```suggestion
          {$_("form.name")}
```

Y declarar la traducción en los dos idiomas que tenemos (`en.json` y `es.json`)